src/pages/ScoreReport.vue

```diff
@@ -883,150 +911,197 @@ const viewOptions = ref([
-          score.correctIncorrectDifference;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Incorrect`] = score.numIncorrect;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Correct`] = score.numCorrect;
+        tableRow[`${taskName} - Correct/Incorrect Difference`] = score.correctIncorrectDifference;
+        tableRow[`${taskName} - Num Incorrect`] = score.numIncorrect;
+        tableRow[`${taskName} - Num Correct`] = score.numCorrect;
       } else if (tasksToDisplayTotalCorrect.includes(taskId)) {
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Attempted`] = score.numAttempted;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Correct`] = score.numCorrect;
+        tableRow[`${taskName} - Num Correct`] = score.numCorrect;
+        tableRow[`${taskName} - Num Attempted`] = score.numAttempted;
       } else if (rawOnlyTasks.includes(taskId)) {
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Raw`] = score.rawScore;
+        tableRow[`${taskName} - Raw`] = score.rawScore;
       } else {
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Percentile`] = score.percentileString;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Standard`] = score.standardScore;
-
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Raw`] = score.rawScore;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Support Level`] = score.supportLevel;
+        tableRow[`${taskName} - Percentile`] = score.percentileString;
+        tableRow[`${taskName} - Standard`] = score.standardScore;
+        tableRow[`${taskName} - Raw`] = score.rawScore;
+        tableRow[`${taskName} - Support Level`] = score.supportLevel;
       }
+
+      // Add reliability information
       if (score.reliable !== undefined && !score.reliable && score.engagementFlags !== undefined) {
         const engagementFlags = Object.keys(score.engagementFlags);
         if (engagementFlags.length > 0) {
           if (includedValidityFlags[taskId]) {
             const filteredFlags = Object.keys(score.engagementFlags).filter((flag) =>
               includedValidityFlags[taskId].includes(flag),
             );
-            if (filteredFlags.length === 0) {
-              tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Reliability`] = 'Unreliable';
-            } else {
-              const engagementFlagString = 'Unreliable: ' + filteredFlags.map((key) => _lowerCase(key)).join(', ');
-              tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Reliability`] = engagementFlagString;
-            }
+            tableRow[`${taskName} - Reliability`] =
+              filteredFlags.length === 0 ? 'Unreliable' : `Unreliable: ${filteredFlags.map(_lowerCase).join(', ')}`;
```

Comment:
No need for `lodash` here either: 

```suggestion
              filteredFlags.length === 0 ? 'Unreliable' : `Unreliable: ${filteredFlags.map(flag => flag.toLowerCase()).join(', ')}`
```

Reply:
Let me double check this first, since I have used `toLowerCase()` before and it has not printed it as I wanted

Reply:
I test it and without _lodash it prints like this: `Unreliable: responsetimetoofast` With _lodash it prints like this: `Unreliable: response time too fast`. So I think for now I will keep the _lodash

Reply:
Oh, yes, that's expected as lodash's `toLowerCase` is case method that inserts spaces – apologies for overlooking that, I expected the flags to be a different format.

```js 
const str = 'ResponseTimeTooFast'

console.log(_.lowerCase(str))  // prints: response time too fast
console.log(_.toLower(str))    // prints: responsetimetoofast
console.log(str.toLowerCase()) // prints: responsetimetoofast
```